### PR TITLE
[VerticalTabsMobile] - add a component

### PR DESCRIPTION
**Pull request by pietro-maximoff**
https://sovryn.atlassian.net/browse/SOV-802

### CODE REVIEW

File: packages/ui/src/2_molecules/VerticalTabsMobile/VerticalTabsMobile.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          </div>
+        </aside>
+      ) : (
+        <div
+          className={classNames(styles.content, props.contentClassName)}
+          ref={contentRef}
+        >
+          <button onClick={handleBack} className={styles.buttonBack}>
+            <Icon icon="arrow-back" size={14} /> Back to wallet menu
+          </button>
```
Comment (creed-victor) [inline packages/ui/src/2_molecules/VerticalTabsMobile/VerticalTabsMobile.tsx]: Text assumes that component will be used for wallets only.

I think entire `button` should be removed from component and it should be added together with `renderContent`, so dev could make sure we can use custom back buttons or even prevent them from appearing.

**Review by creed-victor — CHANGES_REQUESTED** [review]

File: packages/ui/src/2_molecules/VerticalTabsMobile/VerticalTabsMobile.module.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+.content {
+  @apply w-full h-full bg-gray-90 py-6 px-7 relative overflow-y-auto;
+  & button {
+    &:first-child {
+      @apply flex items-center text-sm mb-11;
+      & svg {
+        @apply bg-gray-70 w-6 h-6 p-1 mr-2 rounded flex items-center justify-center;
+      }
+    }
+  }
```
Comment (creed-victor) [inline packages/ui/src/2_molecules/VerticalTabsMobile/VerticalTabsMobile.module.css]: Remove it from here and apply classes inline - as it's only used in storybook it's ok on using tailwind classes directly there.

File: packages/ui/src/2_molecules/VerticalTabsMobile/VerticalTabsMobile.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+          header={() => <p>Select the type of wallet you have</p>}
+          items={[
+            {
+              label: 'Tab 1',
+              content: (
+                <>
+                  <button onClick={() => setSelectedIndex(null)}>
+                    <Icon icon="arrow-back" size={14} />
+                    Back to wallet menu
+                  </button>
```
Comment (creed-victor) [inline packages/ui/src/2_molecules/VerticalTabsMobile/VerticalTabsMobile.stories.tsx]: It's being repeated multiple times, make it a component and reuse it.

**Review by creed-victor — CHANGES_REQUESTED** [review]

**Review by creed-victor — APPROVED** [review]

**Review by soulBit — APPROVED** [review]
Approving but we should replace the `<p>` and `<small>` tags in MobileVerticalTabItem component with `<Heading>`, once Victor has made adjustments to CSS overridesfor font color (or in SOV-1027, whichever comes first)

**Review by LunaSov — APPROVED** [review]

**Pull request merged**